### DISCUSSION TITLE
Hacktoberfest 2022

### DISCUSSION THREAD

**Discussion by shahednasser** [shahednasser/awesome-resources#433] [Announcements]
Thank you for everyone interested in contributing to this repository for Hacktoberfest!

### Hacktoberfest Topic

As this repository already have the `hacktoberfest` topic added to the repository, all your contributions should count into Hacktoberfest. Please do not send me requests to add any additional labels to your PRs unless there's a reason that it will not be merged during October.

### Patience Please

Although I'll do my best to commit more to this repository during Hacktoberfest and not take long to merge PRs, please have some patience. Do not email or message me outside of your PR to merge it in.

### Quality Content Only

It's great you want to participate in Hacktoberfest! However, it's important to remember the purpose of this repository. It's important to provide actual helpful content that can help people using this website, whether during Hacktoberfest or outside Hacktoberfest.

### Types of Contributions

Although this repository's main focus is adding resources, it is also possible to contribute by improving the website, improving the workflow by adding GitHub actions, and more. If you see an opportunity for an enhancement, do not hesitate in sending an issue about it.

### Read Contribution Guidelines

If you're a first-time contributor, please read the [contribution guidelines](https://github.com/shahednasser/awesome-resources/blob/7a2460cfd43c454e1bb84208bb82b6a3cb025fa7/CONTRIBUTING.md) first. It can save time reviewing your PR and help me get through PRs faster.

---

Thank you for everyone contributing!

**Comment by Watrdguy** [comment]
which website can we contribute on?
